README.md

```diff
@@ -12,12 +13,16 @@ client has permission to access resources on behalf of the resource owner.
 
 OAuth2 provides several different methods for the client to obtain
 authorization. At the time of writing, this library only supports the
-[AuthorizationCodeGrant][] method, but further methods may be added in the
-future. The following example uses this method to authenticate, and assumes
-that the library is being used by a server-side application.
+[AuthorizationCodeGrant][] and [resourceOwnerPasswordGrant][] methods, but
+further methods may be added in the future. The following example uses this
+method to authenticate, and assumes that the library is being used by a server-side
+application.
 
-[AuthorizationCodeGrant]: https://api.dartlang.org/apidocs/channels/stable/#oauth2/oauth2.AuthorizationCodeGrant
+[AuthorizationCodeGrant]: http://www.dartdocs.org/apidocs/channels/stable/#oauth2/oauth2.AuthorizationCodeGrant
+[resourceOwnerPasswordGrant]: http://www.dartdocs.org/apidocs/channels/stable/#oauth2/oauth2.resourceOwnerPasswordGrant
```

Comment:
How does dartdocs.org work? Does it scan the pub packages and generate the dartdocs or do you upload the dartdocs to it?

I'm not really sure what to do other than not use links in the README or submit a pull request after the package is published to update the README.   Would you like me to submit a separate pull request after the dartdocs are updated to update the README?